tempodb/backend/raw.go

```diff
@@ -235,16 +235,16 @@ func KeyPathWithPrefix(keypath KeyPath, prefix string) KeyPath {
 }
 
 // MetaFileName returns the object name for the block meta given a block id and tenantid
-func MetaFileName(blockID uuid.UUID, tenantID string) string {
-	return path.Join(RootPath(blockID, tenantID), MetaName)
+func MetaFileName(blockID uuid.UUID, tenantID string, prefix string) string {
+	return path.Join(prefix, tenantID, blockID.String(), MetaName)
 }
 
 // CompactedMetaFileName returns the object name for the compacted block meta given a block id and tenantid
-func CompactedMetaFileName(blockID uuid.UUID, tenantID string) string {
-	return path.Join(RootPath(blockID, tenantID), CompactedMetaName)
+func CompactedMetaFileName(blockID uuid.UUID, tenantID string, prefix string) string {
+	return path.Join(prefix, tenantID, blockID.String(), CompactedMetaName)
 }
 
 // RootPath returns the root path for a block given a block id and tenantid
-func RootPath(blockID uuid.UUID, tenantID string) string {
-	return path.Join(tenantID, blockID.String())
+func RootPath(blockID uuid.UUID, tenantID string, prefix string) string {
```

Comment:
you made a change to this method but then didn't use it? should we continue to use this method above?

Reply:
hi, this method is used by the compactor. https://github.com/grafana/tempo/blob/deeb2636b88bbbc16e288b4c5f744e50bffe20c2/tempodb/backend/s3/compactor.go#L51
this method was also used by the previous `MetaFileName` and `CompactedMetaFileName`
https://github.com/grafana/tempo/blob/f3e3c9a70b370cf1a936451cf2dbcb2404136412/tempodb/backend/raw.go#L244